pyccl/background.py

```diff
@@ -271,35 +272,65 @@ def rho_x(cosmo, a, species, is_comoving=False):
         species_types[species], int(is_comoving))
 
 
-def Sig_MG(cosmo, a):
-    """Redshift-dependent modification to Poisson equation for massless
-    particles under modified gravity.
+def mu_MG(cosmo, a, k=None):
+    """Redshift-dependent modification to Poisson equation under modified
+    gravity.
 
     Args:
-        cosmo (:class:`~pyccl.core.Cosmology`): a Cosmology object.
+        cosmo (:class:`~pyccl.core.Cosmology`): Cosmological parameters.
```

Comment:
I think this is clearer as "a Cosmology object", to be explicit about the type of variable cosmo should be?

Reply:
agreed. Fixed.